ramp4-pcar4/storylines-editor
# remove google sheets, cloud buttons from highcharts editor

### Related Item(s)
#340

### Changes
- Adds some CSS overrides to remove the Google Sheets, Upload/Download from Cloud buttons from the chart editor.

### Testing
Steps:
1. Open the demo page and load a product.
2. Create a new chart and ensure that the "google sheets" import button is gone, and the upload/download from cloud buttons in the top right are gone.
